### DISCUSSION TITLE
OpenAI又搞出了另一個驚人的東西 - ChatGPT

### DISCUSSION THREAD

**Discussion by chia-kai-chang** [1101-GS4719-C/Python-Community-GS4719-C#868] [General]
直接看連結：https://openai.com/blog/chatgpt/

轉載自[Tsung-Hsiang Chang](https://www.facebook.com/vgod.tw?__cft__[0]=AZXkXsc0L5MozmBbel8ElhGx8N5CUbN5Gi0mRjfqbsfx4YJcAWK6PIMfmwVM20snrMrw8lYCDV_8yy61dldM8TJB6QYC5oU_Nydrz-0Z88R2YAPohvzqBY8iG8oOiTKBUKQ&__tn__=-UC%2CP-R)臉書評論：

「它不只能對話，還能解釋複雜的概念。 我剛叫他解釋quicksort給一個五歲小孩聽。我說太難聽不懂，它就換一個方法用更簡單的語言解釋。後來我就開始追問「怎麼知道這方法是對的？」，它一開始只用一些很high level的解釋，像是「這方法已經被廣泛使用和測試過很久了，有很多證據顯示它是對的」。如果繼續追問要它證明，它一開始還會抗拒一下說「證明很難喔，有很多數學。如果你有興趣，可以讀一些研究」。但如果再跟它說「我就是想知道，解釋一個簡單的證明給我聽」，它竟然還真的能給出個不錯的回答來。
Twitter上還有些例子更誇張，貼一段程式碼問它bug在哪，它不但能指出來，還可以跟你說為什麼錯，要怎麼修才對。這一切都是用自然語言對答的。
對了，如果你用中文問ChatGPT它會不會說中文，它會用中文回答「對不起，我不會說中文。我是由OpenAI訓練的語言模型，我的目的是回答問題並提供信息。我只能用英語回答問題。你有什麼其他問題嗎？」 (擺明是睜眼說瞎話)」

[https://twitter.com/....../status/1598269692082151424......](https://twitter.com/bentossell/status/1598269692082151424?s=46&t=tNFHluBUaWBqFKRRizmXQQ&fbclid=IwAR2wSKIeNUe0eZTlq_Ri9tSkkRYZcB2n777LHv9OHmOga-IKot4Z_NrwCWo)
更多嚇死人的範例

大家參考一下~

**Comment by chia-kai-chang** [comment]
奇異點要到了嗎? 抖~ 😱

**Comment by williamyen0623** [comment]
原來AI的功能已經強大到如此地步，不過問他會不會說中文的橋段是真的蠻逗趣的哈哈

**Comment by evanchou1204** [comment]
ChatGPT的功能真的很誇張，自從發現他之後，程式作業再也不用動腦了，都丟給AI做就好了.....

**Comment by yeh-alex** [comment]
最近看到有人把學測國文題目給他做竟然能做到80%以上的準確度，感覺資料量再大一點準確率會很可怕

**Comment by Evan0979** [comment]
越來越像真人了,像到有點令人發毛......

**Comment by CPCpyt** [comment]
我覺得未來仿生機器人的概念將會在未來實行，真的令人驚艷。

**Comment by StevenWanglolz** [comment]
我只知道寫程式遇到不會的問題時問他就好了哈哈哈
